src/models/ReturnSQSEvent.ts

```diff
@@ -6,6 +6,8 @@ export type EventType = "AUTH_IPV_AUTHORISATION_REQUESTED" | "F2F_YOTI_START" |
 export interface ReturnSQSEvent {
 	event_id: string;
 	client_id: string;
+	component_id?: string;
```

Comment:
Is this field unique to `IPV_F2F_USER_CANCEL_END`? 
Wondering why we didn't need it on the interface until now